src/main/kotlin/com/github/davenury/ucac/common/Change.kt

```diff
@@ -3,20 +3,36 @@ package com.github.davenury.ucac.common
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonSubTypes
 import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.github.davenury.ucac.history.History
 import com.github.davenury.ucac.history.HistoryEntry
 import com.github.davenury.ucac.history.InitialHistoryEntry
 import com.github.davenury.ucac.history.IntermediateHistoryEntry
 import com.github.davenury.ucac.objectMapper
 import java.util.*
 import kotlin.collections.ArrayList
 
-typealias History = MutableList<Change>
-
 
 // see https://github.com/FasterXML/jackson-databind/issues/2742#issuecomment-637708397
 class Changes : ArrayList<Change> {
     constructor() : super()
+
     constructor(collection: List<Change>) : super(collection)
+
+    fun toHistory(): History {
+        val h = History()
+        this.map { it.toHistoryEntry() }
+            .forEach { h.addEntry(it) }
+        return h
+    }
```

Comment:
What do you think about change like this?
```suggestion
    fun toHistory(): History = this.fold(History()){h, change -> h.addEntry(change.toHistoryEntry()) }
```

Reply:
My attitude towards this change is ambivalent. However, it does not compile

Reply:
Ok, so add TODO that it should be changed.